packages/chaire-lib-frontend/src/styles/_form.scss

```diff
@@ -64,11 +64,24 @@ textarea, ._input-textarea {
 }
 
 select {
-  font-size: $sm-size;
-  background: $input-background;
+  font-size: $font-size-small;
+  background-color: $input-background;
   color: $input;
   font-weight: 700;
   text-overflow: ellipsis;
+  border: 1px solid $border-white;
+  padding-top: $xxxxs-size;
+  padding-bottom: $xxxxs-size; 
+  padding-left: $xs-size;
+  border-radius: 1.2rem;
+
+  // Unfortunately we can't change the color of the chevron on the select input.
+  // We must add our own. This is a simple Base64-encoded svg image of a white chevron with no background.
+  appearance: none;
+  background-image: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjxzdmcKICAgaWQ9IkxheWVyXzEiCiAgIGRhdGEtbmFtZT0iTGF5ZXIgMSIKICAgdmlld0JveD0iMCAwIDQuOTUgMTAiCiAgIHZlcnNpb249IjEuMSIKICAgc29kaXBvZGk6ZG9jbmFtZT0iaW1hZ2Uuc3ZnIgogICBpbmtzY2FwZTp2ZXJzaW9uPSIxLjMgKDBlMTUwZWQsIDIwMjMtMDctMjEpIgogICB4bWxuczppbmtzY2FwZT0iaHR0cDovL3d3dy5pbmtzY2FwZS5vcmcvbmFtZXNwYWNlcy9pbmtzY2FwZSIKICAgeG1sbnM6c29kaXBvZGk9Imh0dHA6Ly9zb2RpcG9kaS5zb3VyY2Vmb3JnZS5uZXQvRFREL3NvZGlwb2RpLTAuZHRkIgogICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgIHhtbG5zOnN2Zz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyIKICAgeG1sbnM6Y2M9Imh0dHA6Ly9jcmVhdGl2ZWNvbW1vbnMub3JnL25zIyIKICAgeG1sbnM6ZGM9Imh0dHA6Ly9wdXJsLm9yZy9kYy9lbGVtZW50cy8xLjEvIj4KICA8c29kaXBvZGk6bmFtZWR2aWV3CiAgICAgaWQ9Im5hbWVkdmlldzIiCiAgICAgcGFnZWNvbG9yPSIjZmZmZmZmIgogICAgIGJvcmRlcmNvbG9yPSIjMDAwMDAwIgogICAgIGJvcmRlcm9wYWNpdHk9IjAuMjUiCiAgICAgaW5rc2NhcGU6c2hvd3BhZ2VzaGFkb3c9IjIiCiAgICAgaW5rc2NhcGU6cGFnZW9wYWNpdHk9IjAuMCIKICAgICBpbmtzY2FwZTpwYWdlY2hlY2tlcmJvYXJkPSJ0cnVlIgogICAgIGlua3NjYXBlOmRlc2tjb2xvcj0iI2QxZDFkMSIKICAgICBpbmtzY2FwZTp6b29tPSIyMy42IgogICAgIGlua3NjYXBlOmN4PSIyLjQ3ODgxMzYiCiAgICAgaW5rc2NhcGU6Y3k9IjQuOTc4ODEzNiIKICAgICBpbmtzY2FwZTp3aW5kb3ctd2lkdGg9IjE0NDAiCiAgICAgaW5rc2NhcGU6d2luZG93LWhlaWdodD0iNzI4IgogICAgIGlua3NjYXBlOndpbmRvdy14PSIwIgogICAgIGlua3NjYXBlOndpbmRvdy15PSIxMzMiCiAgICAgaW5rc2NhcGU6d2luZG93LW1heGltaXplZD0iMCIKICAgICBpbmtzY2FwZTpjdXJyZW50LWxheWVyPSJMYXllcl8xIiAvPgogIDxkZWZzCiAgICAgaWQ9ImRlZnMxIj4KICAgIDxzdHlsZQogICAgICAgaWQ9InN0eWxlMSI+LmNscy0xe2ZpbGw6I2ZmZjt9LmNscy0ye2ZpbGw6IzQ0NDt9PC9zdHlsZT4KICA8L2RlZnM+CiAgPHRpdGxlCiAgICAgaWQ9InRpdGxlMSI+YXJyb3dzPC90aXRsZT4KICA8cG9seWdvbgogICAgIGNsYXNzPSJjbHMtMiIKICAgICBwb2ludHM9IjEuNDEgNC42NyAyLjQ4IDMuMTggMy41NCA0LjY3IDEuNDEgNC42NyIKICAgICBpZD0icG9seWdvbjEiCiAgICAgc3R5bGU9ImZpbGw6I2ZmZmZmZjtmaWxsLW9wYWNpdHk6MSIgLz4KICA8cG9seWdvbgogICAgIGNsYXNzPSJjbHMtMiIKICAgICBwb2ludHM9IjMuNTQgNS4zMyAyLjQ4IDYuODIgMS40MSA1LjMzIDMuNTQgNS4zMyIKICAgICBpZD0icG9seWdvbjIiCiAgICAgc3R5bGU9ImZpbGw6I2ZmZmZmZjtmaWxsLW9wYWNpdHk6MSIgLz4KICA8bWV0YWRhdGEKICAgICBpZD0ibWV0YWRhdGEyIj4KICAgIDxyZGY6UkRGPgogICAgICA8Y2M6V29yawogICAgICAgICByZGY6YWJvdXQ9IiI+CiAgICAgICAgPGRjOnRpdGxlPmFycm93czwvZGM6dGl0bGU+CiAgICAgIDwvY2M6V29yaz4KICAgIDwvcmRmOlJERj4KICA8L21ldGFkYXRhPgo8L3N2Zz4K);
```

Comment:
ce serait pas préférable de mettre l'image en format png  ou svg dans le repo directement?

Reply:
Oui je peux faire ça!
Ça irait dans transition-frontent ou chaire-lib-frontent?
Pour l'instant je vois qu'il n'y aucun _asset_ dans chaire-lib-frontend. Est-ce un choix délibéré ou est-ce qu'il n'y avait simplement pas le besoin jusqu'à maintenant?
cc @tahini

Reply:
ça irait clairement dans chaire-lib, ce n'est pas spécifique à transition

Reply:
et si possible prendre un svg vectoriel.

Reply:
Bon... c'est plus compliqué que je pensais
Il y a un problème: mon icône ne semble pas être copié dans `public/dist/images/icons/interface`, même si j'ai modifié le fichier de config webpack et le package.json de chaire-lib-frontend pour copier les images au dossier lib.
:-(

Des suggestions? @kaligrafy  @tahini

Reply:
Déplacer dans Transition comme les autrs images, et on fixera ça plus tard. Séparer en issue distinct.

Reply:
done! ready to merge